test/setup_command.bats

```diff
@@ -60,6 +61,78 @@ teardown() {
   grep -F "$EXPECTED_USE_ASDF" "$XDG_CONFIG_HOME/direnv/lib/use_asdf.sh"
 }
 
+@test "setup bash, do not modify rcfile" {
+  run asdf direnv setup --no-touch-rc-file --shell bash --version system
+  # shellcheck disable=SC2016
+  grep -F '${XDG_CONFIG_HOME:-$HOME/.config}/asdf-direnv/bashrc' "$HOME/.bashrc" && return 1 || return 0
```

Comment:
How sure are you that the lines after this actually execute? It seems to me like these `return`s would cause the test to immediately exit? (sorry, I'm on a phone and don't have an easy way to check)

Edit: actually, I'm not sure there's much value in the rest of the test anyways. It looks like it's verifying other aspects of the setup actually happened, but I'd expect we already have tests that confirm that. (similar comment for the rest of these tests)

Reply:
Good catch. I'd copied existing tests to build my tests, and forgot to remove the following redundant lines from each:
```
  grep "export ASDF_DIRENV_BIN" "$XDG_CONFIG_HOME/asdf-direnv/bashrc"
  # shellcheck disable=SC2016
  grep -F 'eval "$($ASDF_DIRENV_BIN hook bash)"' "$XDG_CONFIG_HOME/asdf-direnv/bashrc"
  grep -F "$EXPECTED_USE_ASDF" "$XDG_CONFIG_HOME/direnv/lib/use_asdf.sh"
```
I've made the corrections in all of the added tests.